lib/dom_lite.ml

```diff
@@ -0,0 +1,31 @@
+module Fragment = struct
+  include Core.Fragment
+
+  let make children = make ~children ()
+end
+
+let href str = ("href", str)
+
+let target str = ("target", str)
+
+let className str = ("className", str)
+
+let make_props list : Dom.domProps =
+  Js_of_ocaml.Js.Unsafe.obj
+    ( Array.of_list list
+    |> Array.map (fun (key, value) ->
+           (key, Js_of_ocaml.Js.Unsafe.inject (Js_of_ocaml.Js.string value)) )
```

Comment:
We can skip this `Array.map` by doing this in each attribute constructor function instead.